.github/workflows/preview-listen-fe.yml

```
@@ -51,13 +51,39 @@ jobs:
+          if [[ ! "${{ steps.deploy.outputs.details_url }}" =~ ^https?:// ]]; then
+            echo "Error: Invalid preview URL format"
+            exit 1
+          fi
+      - name: Run Playwright Tests
+        working-directory: apps/listen
+        if: success()
+        run: |
+          pnpm exec playwright install --with-deps
+          pnpm run e2e
+        env:
+          PLAYWRIGHT_TEST_BASE_URL: ${{ steps.deploy_listen.outputs.details_url }}
```

actionlint (1.7.4)

property "deploy_listen" is not defined in object type {deploy: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)